configs/mainnet.yaml

```diff
@@ -29,7 +29,7 @@ GENESIS_DELAY: 604800
 
 # Altair
 ALTAIR_FORK_VERSION: 0x01000000
-ALTAIR_FORK_EPOCH: 18446744073709551615
+ALTAIR_FORK_EPOCH: 74240  # Oct 27, 2021, 10:56:23am UTC
```

Comment:
LGTM. Sanity check: `1606824023+(74240*32*12)=1635332183`, `Wed Oct 27 2021 10:56:23 GMT+0000`.
And extra: `(74240*32)%(2**13)=0` (fresh block/state roots batch)